### PR TITLE
Added presentations page

### PR DESCRIPTION
**Pull request by jebw**
### Trello card

https://trello.com/c/cmJ1aEdI

### Context

We should provide the slides from events online if possible.

### Changes proposed in this pull request

1. Add a presentations page linking to the slides
2. Added (resized) versions of the slides

### Guidance to review

This will be linked from the events pages but that change needs to happen in the main app repo. - for now its accessible at `/presentations` or `/slides`

### CODE REVIEW

**Review by peteryates — DISMISSED** [review]
This looks good to me so I'll approve, but some thoughts while reviewing:

* I wonder if the buttons could be clearer in specifying that the targets are PDF documents
* I _think_ that us serving PDFs might require us to publish an [accessible documents policy](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/accessible-pdfs), not 100% where we stand on that front.

File: content/presentations.md [hunk 1]
```diff
@@ -19,7 +19,8 @@ This presentation covers:
 - Advice and support from Teacher Training Advisers
 
 <a href="/assets/documents/train-to-teach-and-support.pdf" class="call-to-action-button">
-    Train to Teach and Support <span>slides</span>
+    <i class="fas fa-file-pdf"></i>
+    Train to Teach and Support slides
```
Comment (peteryates) [inline content/presentations.md]: Looks like the icon should be after the text here

**Review by peteryates — DISMISSED** [review]
Looks good

File: content/presentations.md (hunk 1 above)
Comment (jebw) [reply content/presentations.md]: Yeh - this was just playing around - pushed by mistake

**Review by peteryates — APPROVED** [review]

**Pull request merged**